bff/src/Duende.Bff.Yarp/DefaultBffYarpTransformerBuilders.cs

```diff
@@ -0,0 +1,33 @@
+// // Copyright (c) Duende Software. All rights reserved.
+// // See LICENSE in the project root for license information.
```

Comment:
Something's off with editorconfig here probably - duplicate //'s.

Reply:
no, was just me manually moving the header back